.github/workflows/ci.yml

```diff
@@ -1,6 +1,8 @@
+---
+
 name: CI
 
-on:
+"on":
```

Comment:
Did a formatter do this? It's the only key that's explicitly a string, which looks a bit weird. Some languages treat on/off as booleans (where this would make more sense), but I don't think YAML is one of them (?).

Reply:
Apparently YAML by standard does treat it as boolean, at least yamllint complained which caused me to change this (see: https://github.com/adrienverge/yamllint/issues/430). Seems that Github implements a custom parser that does not conform to the spec